External/Plugins/AS3Context/Context.cs

```diff
@@ -131,7 +131,7 @@ public Context(AS3Settings initSettings)
             features.privateKey = "private";
             features.intrinsicKey = "extern";
             features.namespaceKey = "namespace";
-
+            features.ArithmeticOperators = new HashSet<char>{'+', '-', '*', '/'};
```

Comment:
What about %, |, ^, etc?

Reply:
I will add all this after this requester, but gradually